pkg/util/objectwatcher/objectwatcher.go

```diff
@@ -1,3 +1,5 @@
+//go:generate mockgen -source=objectwatcher.go  -destination=testing/mock_objectwatcher.go -package=testing ObjectWatcher
```

Comment:
refering to k8s: https://github.com/kubernetes/kubernetes/tree/master/pkg/kubelet/container/testing